python/lsst/ts/eas/diurnal_timer.py

```diff
@@ -49,11 +49,20 @@
 # of sunset should be roughly 2.22 degrees below horizon.
 SOLAR_ELEVATION_AT_SUNSET = -2.22
 
+HOURS_PER_DAY = 24
+SECONDS_PER_HOUR = 3600
+
+CIVIL_TWILIGHT = -6.0
+NAUTICAL_TWILIGHT = -12.0
+ASTRONOMICAL_TWILIGHT = -18.0
+
+TIME_EPSILON = 1 * u.min  # A small increment of time
```

Comment:
This used to be 1 second instead of 1 minute. Are you sure it needs to be 1 minute?